### ISSUE TITLE
Accessibility - Exploitation Types Screen - TalkBack Does Not Read All Goods in Scrollable List

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#50]
TalkBack did not read all the goods in the scrollable list to the right of the country name. An example would be ‘India’ were it read the first fourteen or so goods and not the last eight or so goods. TalkBack needs to be able to read all the items in a list of goods for a given country no matter how many there are.

**Comment by aquilent-dgemoets** [comment]
See discussion for #33.

**Comment by jfc3-dol** [comment]
Not sure how a person using TalkBack would know that they missed other items in the list unless one of the chimes or dings was that notification similar to how when reading links there is a bell chime at the end of the link.

**Comment by jfc3-dol** [comment]
Application still only reads goods that are visually on the screen and not the entire list of items.

**Comment by jfc3-dol** [comment]
Again tested and TalkBack is still not reading goods that are not visually on the screen.

Will attempt to look at this with our newer device tomorrow (Wednesday) to see if it's a problem there too.

**Comment by jfc3-dol** [comment]
Even on our new Samsung Galaxy S7 with newer software TalkBack (screen reader) is not reading the items that are off the screen or even mentioning that there are more items.

**Comment by carolynhuang** [comment]
Cannot resolve this issue without design change.  For readability, the scrolling list design was used so that countries could fit on one line. Please suggest solution for this, as this design feature is incompatible with VoiceOver.

**Comment by jfc3-dol** [comment]
Think we need to rethink a better way to display the longer list of items for the next version of the applications so all users can see all the items. Don't think TalkBack announces the list can be scrolled. If TalkBack did, then the user could possibly scroll the list themselves to see the rest of the list as a stop-gap fix.

**Comment by NJICF** [comment]
added count of goods

**Comment by jfc3-dol** [comment]
Adding the count is a good start to help people using TalkBack to know how many items are there for a given country. Knowing this will allow them to scroll through the list of items manually to get to the ones visually off-screen.

Going to close this issue and open another one for the next version of the software to attempt to come up with a better way of displaying goods.
